contributor.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
```

Comment:
Maybe you'd like to add a hello or something?

"Thank you for taking the time and thinking about contributing. We look forward to your ideas, feedback and PRs!"

Reply:
I love it, thank you very much. I couldn't have written it better. It has been included in the CONTRIBUTING.md